db/version_edit.cc

```diff
@@ -781,6 +801,10 @@ std::string VersionEdit::DebugString(bool hex_key) const {
       r.append(" blob_file:");
       AppendNumberTo(&r, f.oldest_blob_file_number);
     }
+    r.append(" min_timestamp:");
+    r.append(f.min_timestamp);
+    r.append(" max_timestamp:");
+    r.append(f.max_timestamp);
```

Comment:
Sorry for this late comment.
I think printing the timestamps in hex format seems better.

Reply:
OK. now fix it to Slice(f.min_timestamp).ToString(true).